sources/LibProtection.Injections/Caching/CacheFormatItem.cs

```diff
@@ -1,26 +1,33 @@
-﻿namespace LibProtection.Injections.Caching
+﻿namespace LibProtection.Injections
 {
-    public struct CacheFormatItem
+    public struct FormatCacheItem
     {
-        public string Format;
-        public object[] Args;
+        public readonly string Format;
+        public readonly object[] Args;
+
+        public FormatCacheItem(string format, object[] args)
+        {
+            Format = format;
+            Args = args;
+        }
 
         public override bool Equals(object obj)
         {
-            if (obj == null || !(obj is CacheFormatItem) || this.GetHashCode() != obj.GetHashCode()) { return false; }
+            if (!(obj is FormatCacheItem) || GetHashCode() != obj.GetHashCode()) { return false; }
 
-            var that = (CacheFormatItem)obj;
+            var that = (FormatCacheItem)obj;
 
-            if (!Equals(this.Format, that.Format)) { return false; }
```

Comment:
This is result of automatic code style tooling. We are rely on the code style of .NET Core project (https://github.com/dotnet/corefx/blob/master/Documentation/coding-guidelines/coding-style.md), which prescribes to avoid using of this keyword without absolute necessity (see p.4).

But I agree that in this case this word improves readability. Will be fixed before merging.